pkg/logging/logfields/logfields.go

```diff
@@ -23,6 +23,9 @@ const (
 	// NodeName is a human readable name for the node
 	NodeName = "nodeName"
 
+	// NodeEncryptionKey is the IPSec SPI assigned to the associated node.
+	NodeEncryptionKey = "nodeEncryptionKey"
```

Comment:
We already have SPI as a field here. I'd rather stick to one consistent name for this in logs.

Reply:
Thanks, this was just an oversight. Will fix.

Reply:
Fixed in latest push